test/e2e/config/docker.yaml

```diff
@@ -62,7 +62,7 @@ providers:
       new: --metrics-addr=:8080
     files:
     - sourcePath: "../data/shared/v1.0/metadata.yaml"
-  - name: "{go://sigs.k8s.io/cluster-api@v1.5}" # supported release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
+  - name: "{go://sigs.k8s.io/cluster-api@v1.5}" # supported release in the v1beta1 series; this is used for v1alpha3 --> v1beta1 --> main clusterctl upgrades test only.
```

Comment:
Should we drop 1.5? 
(same for other providers)

Reply:
We can't because of the `v1alpha3 => v1beta1 => main` tests.